msgcntr/messageforums-hbm/src/java/org/sakaiproject/component/app/messageforums/dao/hibernate/DraftRecipient.hbm.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE hibernate-mapping PUBLIC "-//Hibernate/Hibernate Mapping DTD 3.0//EN" "http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd">
+<hibernate-mapping package="org.sakaiproject.component.app.messageforums.dao.hibernate">
+
+  <class name="org.sakaiproject.component.app.messageforums.dao.hibernate.DraftRecipientImpl"
```

Comment:
Instead of an HBM please use JPA annotations plenty of examples, shouldn't take much to update.
The addition of **new** hbm files is prohibited, as they're all be converted to JPA.

Reply:
I have never used JPA but I'll give it a shot.